### DISCUSSION TITLE
"Connecting to the RustDesk network", outgoing works fine but incomming dont

### DISCUSSION THREAD

**Discussion by VirusBurst** [rustdesk/rustdesk#10334] [Q&A]
First some important stuff;

I recently changed my router firmware from FritzBox to Freetz-NG and now dnsmasq is doing DHCP, DNS and so.
Since then I run into the issue, that RustDesk on my Windows PC constantly shows "Connecting to the RustDesk network"

And making things funnier i guess, I can connect to any devices but if they try to connect to my PC then it says "The device is offline"

All the ports are still opened and I checked them my phone with mobile data... and even if RustDesk on my PC says it's connecting, I already see which devices are online...

I'm asking for assistant, whether some logs might help to find the issue or it's a local issue on my PC although I probably have to do more than just uninstall and reinstall since the setting were kept intact...

**Comment by rustdesk** [comment]
Go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting or deployment here.
